File: AndroidCourse/Java/Lesson_1_2/app/src/main/java/com/example/lesson_1_2/Student.java
package com.example.lesson_1_2;

public class Student {
    private int id;
    private String name;
    private String surname;
    private String grade;
    private int birthdayYear;

    public Student(int id, String name, String surname, String grade, int birthdayYear) {
        this.id = id;
        this.name = name;
        this.surname = surname;
        this.grade = grade;
        this.birthdayYear = birthdayYear;
    }

    @Override
    public String toString() {
        String textStudent = String.valueOf(this.id) + " " + this.name + " " + this.surname + " " +
                this.grade + " " + String.valueOf(this.birthdayYear);

        return textStudent;
    }
}

File: AndroidCourse/Kotlin/Lesson_1_2/app/src/main/java/com/example/lesson_1_2/Student.kt
package com.example.lesson_1_2

class Student (val id: Int, val name: String, val surname: String, val grade: String, val birthdayYear: Int) {
    override fun toString(): String {
        val outputString = "$id $name $surname $grade $birthdayYear"
        return outputString
    }
}